### ISSUE TITLE
Bring the electronic junk on the floor next to the entrance to the Recyclers

### ISSUE THREAD

**Issue by MetaFight** [OxfordHackspace/HelperWork#10]
There's a pile of junk electronics in the hackspace. It's piled next to the entrance. It's waiting to be brought to the recycling centre.

Are you a bad enough dude to bring it there?

**Comment by NickKolpin** [comment]
How much stuff is there, could you attach a photo?

**Comment by Yanaba** [comment]
Not a whole heck of a lot, but enough to be annoying to step around. Less than one recycling bin's worth (where the bin is one of those low indoor boxy ones with handles for carrying.)

**Comment by uglynewt** [comment]
Some of that stuff was in the ["ready to throw out" pile](http://wiki.oxhack.org/wiki/Clutter) a year ago, in the old space.

Sorry to muddy the waters, but other non-junk items may have ended up in that box during the move.  Last week, we found one of the (still-working) remotes for the LED lights in there.

**Comment by Mellen** [comment]
where is the recycling centre?

**Comment by Yanaba** [comment]
Redbridge I believe.

**Comment by uglynewt** [comment]
Yes, Redbridge.  Opening hours, and what they'll accept, on the county council website:  https://www.oxfordshire.gov.uk/cms/content/find-recycling-centre-0

You have to turn up with a vehicle.  It can be a bicycle, but they won't let you just walk in with an armload of stuff.

**Comment by Yanaba** [comment]
This has mostly been done now, Charles was awesome and took away the vast majority of junk that I spent the evening sorting through and packing into a box. 

Two monitors remain, one of which was donated and I'm told has a relatively minor issue, and the other has a more major issue. If someone could let me know what the minor issue is about and whether that monitor is saveable, then that would be great.

I also saved a pair of heat sinks and a seemingly working mouse with usb adapter. People had packed all kinds of crap into the recyclable electronics box, but things of reasonably conceivable use shouldn't go in there.

**Comment by MetaFight** [comment]
I think useful things were packed into that box during the move from the old space.  The "electronic junk" box got confused with all the boxes being filled up with electronics and people started filling it up with non-junk.

**Comment by Yanaba** [comment]
Ok fair enough.

It's good someone looked through the box and pile before we hauled it off
like this issue wanted--there were many more useful things in there than
advertised, I haven't listed all of them.

In general I think whoever does this recurring recycling job should make
sure they or someone else casts an eye through what's there first.
